src/index.js

```diff
@@ -0,0 +1,116 @@
+const BASE_URL = 'http://localhost:5000';
+
+const state = {};
+
+const landscapes = {
+  '80+': '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂',
+  '70-79': '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷',
+  '60-69': '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃',
+  '59-': '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲',
+};
+
+const skies = {
+  sunny: '☀️☁️☀️ ☁️ ☀️☁️ ☀️ ☁️ ☀️☁️☀️',
+  cloudy: '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️',
+  rainy: '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧',
+  snowy: '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨',
+};
+
+const increaseTemp = (event) => {
+  state.temp++;
+  refreshUI(state.temp);
+};
+
+const decreaseTemp = (event) => {
+  state.temp--;
+  refreshUI(state.temp);
+};
+
+const refreshUI = function () {
+  let tempDisplay = document.getElementById('tempDisplay');
+  let landscapeDisplay = document.getElementById('landscapeDisplay');
+  let skyDisplay = document.getElementById('skyDisplay');
+  let cityNameDisplay = document.getElementById('cityNameDisplay');
+
+  tempDisplay.textContent = state.temp;
+  cityNameDisplay.textContent = state.cityName;
+  skyDisplay.textContent = skies[skySelect.value];
+
+  if (state.temp > 79) {
```

Comment:
I see that you're using an array `landscapes` to help you grab the correct sky you should display. Nice!

How could you update this method to be more data driven instead of relying on several if/else ifs to determine what color to change the temperature text and the landscape?